layouts/fastly/helix.vcl

```diff
@@ -793,6 +792,31 @@ sub hlx_type_proxy {
   return(pass);
 }
 
+/**
+  This checks X-Debug key
+*/
+sub hlx_check_debug_key {
+  declare local var.debugSecret STRING;
+  declare local var.key STRING;
+  declare local var.level STRING;
+
+  set var.debugSecret = table.lookup(secrets, "DEBUG_KEY");
```

Comment:
You could specify a default value here as a third parameter, for instance the service config ID.

Reply:
Ahh for here; this is done in the CLI! the publish always either sends serviceid or some custom debug-key so it will be in the table

Reply:
Sounds good and is probably safer, too.